v2/bolt_store.go

```diff
@@ -266,3 +267,44 @@ func (b *BoltStore) GetUint64(key []byte) (uint64, error) {
 func (b *BoltStore) Sync() error {
 	return b.conn.Sync()
 }
+
+// Transition takes a BoltDB and a path, and outputs a
+// Bbolt and generates it at that path
+func Transition(old *v1.BoltStore, path string) (*BoltStore, error) {
```

Comment:
I did indeedy. At first I tried to get all Options as well, but I am trying to limit the amount of data sharing between the two. We could make another helper function in the v1 repository that outputs all the existing Options so that the new BoltStore could use that 🤔 
What do you think of that?

Reply:
Good point about options.  Though, I wouldn't worry allowing consumers to configure them.  Looking at the [options](https://pkg.go.dev/github.com/boltdb/bolt#Options), I don't anticipate them to be significant when Transition only reads the file in one pass.  Also, looking at Nomad/Consul/Vault, we only use [`raftboltdb.NewBoltStore`](https://github.com/hashicorp/consul/blob/2e4dc7b70539161a40c48076311238adc3eb2835/agent/consul/server.go#L716) without passing any options.